examples/form-choices.rs

```
@@ -0,0 +1,127 @@
+    ];
+
+    form.build_field::<Checkbox>(
+        "checkbox",
+        FormOptions {
+            style: theme,
+            label: Some("Please select something"),
+            custom: HashMap::from([(
+                String::from("choices"),
+                FormValue::List(check_choices.clone()),
+            )]),
+            ..Default::default()
+        },
+    );
+    form.build_field::<Radio>(
+        "radio",
+        FormOptions {
+            style: theme,
+            label: Some("Do you enjoy this demo?"),
+            custom: HashMap::from([(
+                String::from("choices"),
+                FormValue::List(vec![String::from("Yes"), String::from("No")]),
+            )]),
+            ..Default::default()
+        },
+    );
+
+    form.set_active(true);
+
+    while !form.is_finished() {
+        // Poll next event
+        match engine.poll() {
+            // A frame has passed
+            Event::Frame => {
+                engine.clear_screen();
+                engine.print_screen(1, 1, form.draw((engine.frame_count % 8 > 3) as usize));
+                engine.draw();
+            }
+
+            // exit with Escape
+            Event::Key(KeyEvent {
+                code: KeyCode::Esc,
+                modifiers: _,
+            }) => {
+                break;
+            }
+            // exit with CTRL+C
+            Event::Key(KeyEvent {
+                code: KeyCode::Char('c'),
+                modifiers: KeyModifiers::CONTROL,
+            }) => {
+                break;
+            }
+            // Let the form handle the unhandled events
+            event => form.handle_event(event),
+        }
+    }
+
+    // we don't need the engine anymore, dropping it will close the fullscreen mode and bring us back to our terminal
+    drop(engine);
+
+    if form.is_finished() {
+        // Get the output of each fields
+        if let FormValue::Vec(selection_list) = form.get_result("checkbox").unwrap_or_default() {
```

I feel like using `unwrap_or_default` can potentially silence any errors that would occur.
Generally, I'm against using `unwrap_or_default` on `Result`s. On `Option`s it makes more sense.

---

I agree with you on that point, it was a little side effect from changing the return value of `get_result`.